### PR TITLE
Added Shotcut video editor description in list

### PR DESCRIPTION
**Pull request by JoseJacsan**

### CODE REVIEW

File: content/media-production-on-pop.md [hunk 1]
```diff
@@ -89,8 +90,17 @@ Install Directly From the Pop!\_Shop
...
 ### Flowblade
 
 Flowblade is a multitrack non-linear video editor with a set of industry standard editing tools suitable for beginners and masters alike to help make your vision a reality of image and sound.
 
-Install Directly From the Pop!\_Shop
+Install Directly From the Pop!_Shop
+
+### Shotcut 
+
+Shotcut is a free, open source, cross-platform video editor. Edit your videos with this lightweight editor full.
```
Comment (jacobgkau) [inline content/media-production-on-pop.md:103]: > Edit your videos with this lightweight editor full.

This sentence doesn't make sense; what does `full` at the end of the sentence mean? (It sounds like it's cut off, e.g. "full of advanced features" or something.)

**Review by jacobgkau — CHANGES_REQUESTED** [review]
Thank you for the contribution. With numerous MLT-based editors available (at least Flowblade, Shotcut, and Kdenlive), I wonder if we'll need to differentiate the feature sets a bit further in order to avoid overwhelming people with too many similar-sounding options. We might be okay for now if the new Shotcut description can be fixed, though.

File: content/media-production-on-pop.md (hunk 1 above)
Comment (JoseJacsan) [reply content/media-production-on-pop.md:103]: > > Edit your videos with this lightweight editor full.
> 
> This sentence doesn't make sense; what does `full` at the end of the sentence mean? (It sounds like it's cut off, e.g. "full of advanced features" or something.)

Sorry, my eng is a little bad

File: content/media-production-on-pop.md [hunk 2]
```diff
@@ -89,8 +90,17 @@ Install Directly From the Pop!\_Shop
...
 ### Flowblade
 
 Flowblade is a multitrack non-linear video editor with a set of industry standard editing tools suitable for beginners and masters alike to help make your vision a reality of image and sound.
 
-Install Directly From the Pop!\_Shop
+Install Directly From the Pop!_Shop
+
+### Shotcut 
+
+Shotcut is a free, open source, cross-platform video editor. Edit your videos with this lightweight editor.
```
Comment (jacobgkau) [inline content/media-production-on-pop.md:103]: `lightweight` is really the only new word that the second sentence is adding here.

Comment (jacobgkau) [reply content/media-production-on-pop.md]: I've replaced the second sentence with one that adds a little more value.

**Review by jacobgkau — DISMISSED** [review]

**Review by ahoneybun — APPROVED** [review]
:tada:

**Review by jacobgkau — DISMISSED** [review]
The only change from my previous approval is removing trailing whitespace.

**Review by jacobgkau — APPROVED** [review]

**Comment by jacobgkau** [comment]
The lint is passing now.

**Pull request merged**